src/components/Computer/Computer.module.css

```diff
@@ -1,45 +1,138 @@
 .computer {
-  z-index: 1;
-  width: 100%;
-  max-width: 300px;
-}
-.monitor {
-  width: 100%;
-  background-color: #eee;
-  border-radius: 0.5rem;
-}
-.screen {
-  height: 100%;
-  padding: 1rem;
-}
-.innerScreen {
-  height: 100%;
-  overflow: hidden;
-  border-radius: 0.5rem;
-  border: 2px solid #000;
-  aspect-ratio: 4/3;
-  background-color: #000;
-  position: relative;
-}
-.buttons {
-  height: 30px;
-  background-color: #eee;
-  border-top: 5px solid #000;
-  border-radius: 0 0 0.5rem 0.5rem;
-  display: flex;
-  gap: 0.5rem;
-  padding: 0.5rem;
-}
+	width: 300px;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	margin: 0 auto;
 
-.fan {
-  background-color: #000;
-  width: 3px;
-}
+	.monitor {
+		width: 95%;
+		height: 200px;
+		background-color: #d0d0d0;
+		border: solid 4px #a0a0a0;
+		border-radius: 10px;
+		padding: 15px;
+		position: relative;
+
+		.inner_screen {
+			width: 100%;
+			height: 100%;
+			background-color: #696969;
+			display: flex;
+			border-radius: 5px;
+			box-sizing: border-box;
+			padding: 10px 15px;
+
+			.screen {
+				width: 85%;
+				height: 100%;
+				border-radius: 5px;
+				background-color: #000080;
+				color: white;
+			}
+
+			.buttons {
+				width: auto;
+				height: 100%;
+				display: flex;
+				flex-direction: column;
+				box-sizing: border-box;
+				border-left: 3px solid #a0a0a0;
+				padding: 10px;
+				margin-left: 15px;
+
+				.button_top {
+					width: 10px;
+					height: 10px;
+					background-color: #ffffff;
+					margin-top: 5px;
+				}
+
+				.button_bottom {
+					width: 10px;
+					height: 10px;
+					background-color: #000000;
+					margin-bottom: 5px;
+				}
+			}
+		}
+	}
+
+	.cpu {
+		width: 115%;
```

Comment:
If you check on mobile viewports this is making the computer break outside the width of the screen. Maybe make it 100%? and add some padding so the computer doesn't touch the border of the screen. Great job!

![Screenshot 2024-11-06 at 7 52 19 PM](https://github.com/user-attachments/assets/a10b72f2-cd15-40dd-b362-1245ed82311e)